StarkInfra/StarkInfra/IssuingEmbossingKit/IssuingEmbossingKit.cs

```diff
@@ -0,0 +1,229 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using StarkInfra.Utils;
+
+
+namespace StarkInfra
+{
+    /// <summary>
+    /// IssuingEmbossingKit object
+    /// <br/>
+    /// The IssuingEmbossingKit object displays information on the card and card package designs available to your Workspace.
+    /// <br/>
+    /// Properties:
+    /// <list>
+    ///     <item>ID [string]: unique id returned when IssuingEmbossingKit is created. ex: "5656565656565656"</item>
+    ///     <item>Name [string]: card or package design name. ex: "stark-plastic-dark-001"</item>
+    ///     <item>Designs [list of IssuingDesign objects]: list of IssuingDesign objects. ex: new List<IssuingDesign> {IssuingDesign(), IssuingDesign() }</item>
+    ///     <item>Updated [DateTime]: latest update datetime for the IssuingEmbossingKit. ex: DateTime(2020, 3, 10, 10, 30, 0, 0)</item>
+    ///     <item>Created [DateTime]: creation datetime for the IssuingEmbossingKit. ex: DateTime(2020, 3, 10, 10, 30, 0, 0)</item>
+    /// </list>
+    /// </summary>
+    public partial class IssuingEmbossingKit : Resource
+    {
+        public string Name { get; }
+        public List<IssuingDesign> Designs { get; }
+        public DateTime? Updated { get;  }
+        public DateTime? Created { get; }
+
+        /// <summary>
+        /// IssuingEmbossingKit object
+        /// <br/>
+        /// The IssuingEmbossingKit object displays information on the card and card package designs available to your Workspace.
```

Comment:
The IssuingEmbossingKit object displays information on the embossing kits available to your Workspace.